### ISSUE TITLE
Non-responsiveness due to posframe focus

### ISSUE THREAD

**Issue by TheJJ** [tumashu/company-posframe#61]
Sometimes, I haven't figured out why, point focus moves inside the posframe.
Then, no keybindings work any more, and the posframe buffer can directly be edited (you can insert chars and delete them etc).

The only way out is clicking by mouse on the real window where you were typing.

Is this a known problem and if yes, how do I prevent it from happening?

**Comment by tumashu** [comment]
yes, posframe will move mouse out posframe

**Comment by TheJJ** [comment]
So you mean this should happen and the mouse focus should automatically be ejected from the posframe?

**Comment by tumashu** [comment]
no, banish mouse and let mouse can not stay on the posframe. so it can not be focus.

**Comment by TheJJ** [comment]
I'm not sure I understand - my mouse got captured in the posframe without any click by me. I was just typing and the mouse is somewhere else on the screen, but then sometimes it gets automatically captured without me clicking or performing mouse movements.
